src/main/java/com/hyperwallet/clientsdk/util/MultipartUtility.java

```diff
@@ -0,0 +1,119 @@
+package com.hyperwallet.clientsdk.util;
+
+import cc.protea.util.http.Response;
+import com.hyperwallet.clientsdk.HyperwalletException;
+
+import javax.xml.bind.DatatypeConverter;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.file.Files;
+
+public class MultipartUtility {
```

Comment:
I'm confused about this implementation. In my opinion, Multipart should be a simple builder class (similar to what [Jersey does](https://github.com/jersey/jersey/blob/master/media/multipart/src/main/java/org/glassfish/jersey/media/multipart/FormDataMultiPart.java)).

We need a clear separation between:
- Hyperwallet (API entry point)
- HyperwalletApiClient.java (manages the HTTP request creation and the HTTP response)
- Multipart (as a parameter for the API entry point call)

Reply:
resolved